com.inochi2d.inochi-creator.yml

```diff
@@ -0,0 +1,83 @@
+id: com.inochi2d.inochi-creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+# sdk-extensions:
+#   - org.freedesktop.Sdk.Extension.ldc
+#   - org.freedesktop.Sdk.Extension.dmd #This lacks aarch64 support, so gonna just pull a binary and later use only-arches to grab one when/if it's released.
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc" # The X and Wayland sockets are permanently exposed, because kdialog takes longer to start if it only has X access, and doesn't work at all if only using Wayland.
+    - "--socket=wayland"

```

Comment:
Does it make sense to include Wayland at all?

Reply:
Kdialog itself runs natively, and I remember having some issues when revoking Wayland access as well. It's weird. Hopefully it's fixed in the next release, since that implements a better way to check for which session it's using.

Reply:
here's the improved session detection, in case you wanted to take a peek: https://github.com/Inochi2D/inochi-creator/commit/62099f3c6228a34c1d0127abd51a78802d5bddde